migrations_lockfile.txt

```diff
@@ -8,6 +8,5 @@ will then be regenerated, and you should be able to merge without conflicts.
 feedback: 0003_feedback_add_env
 nodestore: 0002_nodestore_no_dictfield
 replays: 0003_add_size_to_recording_segment
-sentry: 0571_add_hybrid_cloud_foreign_key_to_slug_reservation
+sentry: 0572_add_first_seen_index_groupedmessage
 social_auth: 0002_default_auto_field
-hybridcloud: 0004_add_cache_version
```

Comment:
Could we keep this line? I'll get the generation tool fixed as this line has been removed in a few pull requests and needed to be re-added manually each time.

Reply:
Thanks for catching this and merging in the fix!